src/utils/textContextUtils.ts

```diff
@@ -0,0 +1,26 @@
+export const aktorHarOpprettetElement = (
+  innloggetFnr: string | null | undefined,
+  arbeidstakerFnr: string,
```

Comment:
Dersom innloggetFnr kan være null eller undefined vil denne gi false, selv om den kanskje egentlig burde bli true. (F.eks finner ikke tidligere arbeidsgiver). 

Jeg tror kanskje det derfor er sikrere å bruke isAudienceSykmeldt, for da trenger man ikke forholde seg til innloggetFnr, siden arbeidstakerFnr ikke er nullable.

ala

```
if (isAudienceSykmeldt) {
      return opprettetAvFnr === arbeidstakerFnr;
    }
    return opprettetAvFnr !== arbeidstakerFnr;
```